rllib/offline/estimators/direct_method.py

```diff
@@ -16,44 +17,58 @@
 
 torch, nn = try_import_torch()
 
+logger = logging.getLogger()
 
-# TODO (rohan): replace with AIR/parallel workers
-# (And find a better name than `should_train`)
-@DeveloperAPI
-def k_fold_cv(
-    batch: SampleBatchType, k: int, should_train: bool = True
-) -> Generator[Tuple[List[SampleBatch]], None, None]:
-    """Utility function that returns a k-fold cross validation generator
-    over episodes from the given batch. If the number of episodes in the
-    batch is less than `k` or `should_train` is set to False, yields an empty
-    list for train_episodes and all the episodes in test_episodes.
 
+@ExperimentalAPI
+def train_test_split(
+    batch: SampleBatchType,
+    train_test_split_val: float = 0.0,
+    k: int = 0,
+) -> Generator[Tuple[List[SampleBatch]], None, None]:
+    """Utility function that returns either a train/test split or
```

Comment:
Nit: Our docstrings should always just have a single sentence one-line at the top, then an empty line, then any other information. E.g.:

```
def xyz(..):
    """Some single sentence description.

    Anything else blabla

    Args:
        ...
    """
```